variables.tf

```diff
@@ -37,3 +37,32 @@ variable "tags" {
   type    = "map"
   default = {}
 }
+
+variable "privileged_mode" {
+  default     = false
+  description = "(Optional, used when building Docker images). If set to true, enables running the Docker daemon inside a Docker container."
+}
+
+variable "aws_region" {
+  type        = "string"
+  default     = ""
+  description = "(Optional, used as ENV variable when building Docker images and pushing them to ECR). AWS Region for the ECR repository, e.g. us-east-1. For more info: http://docs.aws.amazon.com/codebuild/latest/userguide/sample-docker.html"
+}
+
+variable "aws_account_id" {
+  type        = "string"
+  default     = ""
+  description = "(Optional, used as ENV variable when building Docker images and pushing them to ECR). AWS Account ID that owns the ECR repository. For more info: http://docs.aws.amazon.com/codebuild/latest/userguide/sample-docker.html"
```

Comment:
I don't think this needs to be ECR specific. This is just a generic variable the communicates the account the build process is operating in.